src/mo_def/syntax.ml

```diff
@@ -33,6 +33,7 @@ type func_sort = Type.func_sort Source.phrase
 
 type mut = mut' Source.phrase
 and mut' = Const | Var
+and idm = (string, mut') Source.annotated_phrase
```

Comment:
Maybe a better name would be id_occ for identifier occurence, or id_ref for identifier reference?

Reply:
Sure! c007a8a3ebc640fd9a35acc32c0666450dfd3da7